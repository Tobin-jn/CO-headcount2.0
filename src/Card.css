.card {
  border: thin solid black;
  padding: 15px;
  margin: 10px;
  width: 17%;
  font-family: 'Raleway', sans-serif;
  box-sizing: border-box;

}

.selected {
  width: 25%;
}

.card-heading {
  font-weight: 600;
  font-size: 1em;
  margin-bottom: 5px;
  /*text-align: center;*/
}

.sub-heading {
  font-size: .8em;
  font-weight: 600;
  margin-bottom: 5px;
}

.card-data {
  color: gray;
  /*text-indent: 30%;*/
}

.year-data {
  margin: 0;
}

.greater-than {
  color: #0095A9;
  font-weight: 600;
}

.less-than-tag {
  margin-top: 10px;
  font-size: .8em;
}