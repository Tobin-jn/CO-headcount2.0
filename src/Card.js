import React from 'react'
import PropTypes from 'prop-types'

const Card = ({district}) => {
  
  const dataPoints = district.stats.map( year => {
    return <p className='year-data' key={Math.random()}> {year.year}: {year.data} </p>

  })

  return(
    <div className='card'>
      <h1 className='card-heading'>{district.location}</h1>
      <div className='card-data'>
        {dataPoints}
      </div>
    </div>

  )
}

Card.propTypes = {
  district: PropTypes.object.isRequired,
  
}

export default Card