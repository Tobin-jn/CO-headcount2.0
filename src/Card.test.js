import React from 'react'
import { shallow, mount } from 'enzyme'

import Card from './Card'

describe('Card', () => {

  it('should match the snapshot with all data passed in correctly', () => { //change to say what data
    const wrapper = shallow(<Card 
                              propName='propname'
                            />)
    expect(wrapper).toMatchSnapshot()
  })
})
