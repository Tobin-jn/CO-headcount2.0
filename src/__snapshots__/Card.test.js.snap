// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot with all data passed in correctly 1`] = `
<div
  className="card"
>
  <h1
    className="card-heading"
  >
    Colorado
  </h1>
  <div
    className="card-data"
  >
    <p
      className="year-data"
      key="1538711787341"
    >
       
      data
      : 
       
    </p>
  </div>
</div>
`;
